charts/telepresence/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "telepresence.fullname" . }}
+  labels:
+    {{- include "telepresence.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "telepresence.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "telepresence.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
```

Comment:
When users are in an air-gapped environment, (thus needing a license) they use `telepresence license -f <file>` it outputs a secret like so: 
```
apiVersion: v1
data:
  hostDomain: <auth.datawire.io (encoded in base64)>
  license: <base64 encoded jwt>
kind: Secret
metadata:
  creationTimestamp: null
  name: systema-license
  namespace: ambassador
  ```
  The only value above that will change for users is the license, since the hostDomain only other suitable value is for staging.
  
  In the yaml when the systema-license secret is found in the ambassador namespace, we add the following:
  ```
          volumes:
        - name: license
          secret:
            defaultMode: 420
            secretName: systema-license
```

and:
```
          volumeMounts:
          - mountPath: /home/telepresence/
            name: license
            readOnly: true
```
The name of the secret can probably be changed if the user wants but the `mountPath` + `name` in the volume mount are explicitly used in the traffic-manager so they need to remain those values.

let me know if any of this doesn't make sense.. more info on the license workflow can be found at the bottom of this page: https://www.getambassador.io/docs/telepresence/latest/reference/cluster-config/

Reply:
Hmmm okay so it sounds like you are suggesting that we create this `volumeMount` by default in the chart and have the user add the license later if they need to manually.

Is that correct?